### DISCUSSION TITLE
Koordinatenabfrage - Ausdruck Koordinatenmarker beschriften

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#112] [Ideas]
![image](https://github.com/gis-eni/webgis-forum/assets/124671123/9a546029-018a-481a-b27e-6b718f40f0f2)

Hier wäre es gut, wenn man die ein oder andere Angabe zusätzlich auswählen könnte und nicht auf eine beschränkt ist. 
Vielleicht eine beschränkte Auswahl um die Lesbarkeit zu gewährleisten bzw. hängt ja das schlussendlich vom Abstand u der Anzahl der Punkte ab u könnte der Anwender selbst entscheiden.

Oder die Möglichkeit in der Konfiguration z.B. custom.json die Anzahl der auswählbaren Beschriftungen individuell einzustellen.

**Comment by gis-eni** [comment]
Wenn man hier mehre Koordinaten Systeme anbietet, ist die Karte schnell einmal mit Werten voll gepflastert und man kann nichts mehr lesen. Ich halte davon eher nichts, weil man mit solchen Funktionen nie fertig wird: 

Wie du schon bemerkst hast, müsste man die Anzahl sowieso irgendwie beschränken. Wenn Punkte nahe beieinander liegen kann man nichts mehr lesen. usw. Außerdem würde ich als Anwender verzweifeln, wenn unterschiedliche Koordinaten für jeden Punkt in der Karte stehen würden. Mir ist der Anwendungsfall nicht klar. Irgendwann muss ich die Koordinaten ja für irgendwas verenden/abtippen. Da kommt es sicher zu Verwechslungen...

Wenn man wirklich mehre Koordinatensystem braucht ist es wohl besser, sich diese zusätzlich über ein CSV mit herunter zu laden. ZB in Karte der stehen die WGS 84 Koordinaten und im CSV nehm ich mir noch die GK Koordinaten (mit den Höhen) dazu.
Wenn ich CSV mehre System brauche, kann ich den Ausdruck öfter durchführen... Ich frage mich aber, wie oft kommt sowas vor?

**Comment by ue-black** [comment]
Um die Anfrage zu präzisieren: Bei unserer Anfrage ging es nicht um die Anzeige mehrerer KBSe. Die meisten arbeiten in einem KBS. Vielmehr ging es darum Gelände UND Oberfläche anzuzeigen. Im 4er sah das so aus:
![image](https://github.com/gis-eni/webgis-forum/assets/127403099/0555a105-d143-4e4a-90fd-486c19ec84f3)

Mock-Up:
![image](https://github.com/gis-eni/webgis-forum/assets/127403099/f1a39c4d-1de2-4976-a295-d55403095880)

Oder eine Kombination aus einem KBS + Gelände/Oberfläche.

**Comment by gis-eni** [comment]
Idee: In der Auswahlliste für die Beschriftungsfelder im Ausdruck sollten alle Kombinationen von Höhenmodellen angezeigt.

**Comment by gis-eni** [comment]
Ab Build 5.23.4501:

Es werden alle möglichen 2x Kombinationen der Höhen für die Ausdruck angeboten:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/6e1fa44a-4652-4435-bf05-6becf9729d83)

Ergebnis in der Karte:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/71e6be45-c1c4-4a1d-a9b9-cf89e79650ce)
